packages/core-mobile/app/contexts/SendContext.tsx

```diff
@@ -0,0 +1,127 @@
+import React, {
+  createContext,
+  Dispatch,
+  ReactNode,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState
+} from 'react'
+import { NetworkTokenUnit, Amount } from 'types'
+import { useNetworks } from 'hooks/networks/useNetworks'
+import { useNetworkFee } from 'hooks/useNetworkFee'
+import { TokenWithBalance } from '@avalabs/vm-module-types'
+
+interface SendContextState {
+  token: TokenWithBalance | undefined
+  setToken: Dispatch<TokenWithBalance | undefined>
+  amount: Amount | undefined
+  setAmount: Dispatch<Amount>
+  toAccount: Account
+  setToAccount: ({ address, title }: { address: string; title: string }) => void
```

Comment:
feel like it's a bit redundant to have both setToAccount and toAccount of type Account here. there are setTitle and setAddress inside toAccount.

Reply:
I realized that we are not using the title from anywhere, so renamed the state to `toAddress` and adjusted the code.